apps/hash-frontend/src/pages/[shortname]/[page-slug].page.tsx

```diff
@@ -373,23 +373,21 @@ const Page: NextPageWithLayout<PageProps> = ({
 
   return (
     <>
-      <Head>
-        <title>{pageTitle || "Untitled"} | HASH</title>
-
-        {/*
-          Rendering favicon.png again even if it's already defined on _document.page.tsx,
-          because client-side navigation does not fallback to the default icon when visiting a page without an icon
-        */}
```

Comment:
I removed it because we're no longer doing what is described (including the default `favicon.png` link if no `icon` is set for the page) – the issue with the browser not refreshing the favicon properly is handled in the layout instead, [here](https://github.com/hashintel/hash/blob/e2a0cdecb2791b5b8b5d7ef2a17540714937e04f/apps/hash-frontend/src/shared/layout/plain-layout.tsx#L48)